tests/test_addon_detail.py

```diff
@@ -286,7 +286,7 @@ def test_more_info_addon_size(selenium, base_url, variables):
     # get the file URL and read its size - conversion from bytes to Mb is required
     file = urllib.request.urlopen(
         'https://addons.allizom.org/firefox/downloads/file/1097275/ghostery_privacy_ad_blocker-8.5.6-an+fx.xpi')
-    size = file.length/(1024*1024)
+    size = file.length / (1024 * 1024)
```

Comment:
I don't see the need for this change. Any reason why you added these spaces?

Reply:
Spaces removed. I think they have been added by the automatic formatter used by my IDE.